CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## [Preview Release 3.0.0-preview2.21105.1] - 2021-04-15
+
+### Breaking Changes over preview release v3.0.0-preview1
+- `User Id` connection property now accepts `Client Id` instead of `Object Id` for **User-Assigned Managed Identity** [#1010](https://github.com/dotnet/SqlClient/pull/1010)
```

Comment:
I'd say requires.
```suggestion
- `User Id` connection property now requires `Client Id` instead of `Object Id` for **User-Assigned Managed Identity** [#1010](https://github.com/dotnet/SqlClient/pull/1010)
```